pyaerocom/test/test_colocation_auto.py

```diff
@@ -0,0 +1,181 @@
+import os
+import pytest
+
+from pyaerocom.conftest import TESTDATADIR
+from pyaerocom.conftest import does_not_raise_exception
+from pyaerocom.conftest import testdata_unavail
+
+from pyaerocom import Colocator, ColocatedData, GriddedData, UngriddedData
+from pyaerocom.io import ReadGridded
+from pyaerocom.exceptions import DataCoverageError
+from pyaerocom.io.aux_read_cubes import add_cubes
+
+
+@pytest.fixture(scope='module')
+def init_obsdata_paths():
+    import pyaerocom as pya
+    pya.const.add_data_search_dir(TESTDATADIR.joinpath('obsdata'))
```

Comment:
What is the advantage of having this as a fixture, instead of just adding this search directory in conftest?

Reply:
None that I can think of, will move it!